### PR TITLE
[#noissue] Polishing

### PR DESCRIPTION
**Pull request by emeroad**

### CODE REVIEW

File: plugins/reactor-netty/src/main/java/com/navercorp/pinpoint/plugin/reactor/netty/interceptor/HttpClientOperationsOnInboundNextInterceptor.java [hunk 1]
```diff
@@ -81,7 +82,7 @@ public void doInAfterTrace(SpanEventRecorder recorder, Object target, Object[] a
     }
 
     private boolean validate(final Object[] args) {
-        if (args == null || args.length < 2) {
+        if (ArrayUtils.getLength(args) != 1) {
```
Comment (koo-taejin) [inline plugins/reactor-netty/src/main/java/com/navercorp/pinpoint/plugin/reactor/netty/interceptor/HttpClientOperationsOnInboundNextInterceptor.java:85]: I think `args.length < 2` would be better to change  `ArrayUtils.getLength(args) < 2`.

File: plugins/reactor-netty/src/main/java/com/navercorp/pinpoint/plugin/reactor/netty/interceptor/HttpServerHandleHttpServerStateInterceptor.java [hunk 2]
```diff
@@ -34,7 +35,7 @@ public HttpServerHandleHttpServerStateInterceptor(TraceContext traceContext, Met
     }
 
     public boolean validate(Object[] args) {
-        if (args == null || args.length < 2) {
+        if (ArrayUtils.getLength(args) != 1) {
```
Comment (koo-taejin) [inline plugins/reactor-netty/src/main/java/com/navercorp/pinpoint/plugin/reactor/netty/interceptor/HttpServerHandleHttpServerStateInterceptor.java:38]: I think `args.length < 2` would be better to change `ArrayUtils.getLength(args) < 2`, args could be 0 size arrays.

File: plugins/reactor-netty/src/main/java/com/navercorp/pinpoint/plugin/reactor/netty/interceptor/HttpServerHandleStateInterceptor.java [hunk 3]
```diff
@@ -33,7 +34,7 @@ public HttpServerHandleStateInterceptor(TraceContext traceContext, MethodDescrip
     }
 
     public boolean validate(Object[] args) {
-        if (args == null || args.length < 2) {
+        if (ArrayUtils.getLength(args) != 1) {
```
Comment (koo-taejin) [inline plugins/reactor-netty/src/main/java/com/navercorp/pinpoint/plugin/reactor/netty/interceptor/HttpServerHandleStateInterceptor.java:37]: I think `args.length < 2` would be better to change `ArrayUtils.getLength(args) < 2`, args could be 0 size arrays.

File: plugins/tomcat/src/main/java/com/navercorp/pinpoint/plugin/tomcat/interceptor/RequestStartAsyncInterceptor.java [hunk 4]
```diff
@@ -72,8 +74,9 @@ public void after(Object target, Object[] args, Object result, Throwable throwab
             final SpanEventRecorder recorder = trace.currentSpanEventRecorder();
             if (validate(target, result, throwable)) {
                 com.navercorp.pinpoint.bootstrap.context.AsyncContext nextAsyncContext = recorder.recordNextAsyncContext(true);
-                if (args != null && args.length > 1 && args[0] instanceof AsyncContextAccessor) {
-                    ((AsyncContextAccessor) args[0])._$PINPOINT$_setAsyncContext(nextAsyncContext);
+                Object asyncContextAccessor = ArrayUtils.get(args, 0);
+                if (asyncContextAccessor instanceof AsyncContextAccessor) {
+                    ((AsyncContextAccessor) asyncContextAccessor)._$PINPOINT$_setAsyncContext(nextAsyncContext);
```
Comment (koo-taejin) [inline plugins/tomcat/src/main/java/com/navercorp/pinpoint/plugin/tomcat/interceptor/RequestStartAsyncInterceptor.java]: When args has a size of 1, the old code is not executed, but changed code is going to be executed.
I don't know if the size of args can be 1.

File: plugins/reactor-netty/src/main/java/com/navercorp/pinpoint/plugin/reactor/netty/interceptor/HttpServerHandleHttpServerStateInterceptor.java (hunk 2 above)
Comment (emeroad) [reply plugins/reactor-netty/src/main/java/com/navercorp/pinpoint/plugin/reactor/netty/interceptor/HttpServerHandleHttpServerStateInterceptor.java:38]:
```java
      if (!(args[1] instanceof HttpResponse)) {
            // Skip LastHttpContent or else.
            return false;
        }
```
If you look at the meaning of the whole code, `getLength(args) != 1` seems to be more accurate.

**Review by jaehong-kim — APPROVED** [review]

**Pull request merged**